app/views/layouts/application.html.haml

```diff
@@ -2,7 +2,7 @@
 %html{lang: "en"}
   %head
     %meta{charset: "utf-8"}/
-    %meta{content: "width=device-width, initial-scale=1.0", name: "viewport"}/
+    %meta{content: "width=device-width, initial-scale=1", name: "viewport"}/
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace